### DISCUSSION TITLE
字段长度超出，不报错？

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#12158] [A - General / Q&A] [answered]
定义了一个 字段 长度为varchar(20)， 但是我写入的数据长度为1000，不报错 也没有任何反应？？？    字段多多情况下，这样很难排查出哪个字段长度超出了。

**Comment by stalary** [comment, marked as answer]
You can set `enable_insert_strict=true` when you insert data.


by stalary


在2022年08月29日 ***@***.***> 写道：

定义了一个 字段 长度为varchar(20)， 但是我写入的数据长度为1000，不报错 也没有任何反应？？？ 字段多多情况下，这样很难排查出哪个字段长度超出了。
